#! /bin/sh -e
## 20_module_mk_shared.dpatch by Craig Small <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Made SHARED variable overwritable

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argum
ent"
                exit 1;;
esac

exit 0
@DPATCH@
--- procps-3.2.0.orig/proc/module.mk
+++ procps-3.2.0/proc/module.mk
@@ -3,7 +3,7 @@
 # for lib$(NAME).so and /usr/include/($NAME) and such
 NAME      :=  proc
 
-SHARED := 1
+SHARED ?= 1
 
 SONAME    :=  lib$(NAME).so.$(LIBVERSION)
 
